### PR TITLE
Added function divisible_by

### PR DESCRIPTION
**Pull request by OlyaKhomyn**

### CODE REVIEW

**Review by iPavliv — DISMISSED** [review]

**Review by khadai — CHANGES_REQUESTED** [review]
Where is main.py?

**Review by khadai — DISMISSED** [review]

**Comment by OlyaKhomyn** [comment]
> Where is main.py?

Fixed

**Review by ement06 — DISMISSED** [review]

File: tasks/kyu8.py [hunk 1]
```diff
@@ -15,3 +15,19 @@ def example(first, second):
...
+    """
+        Returns a list of numbers which are divisible by given number
+        Args:
+            array (list): The first parameter
+            divisor (int): The second parameter
+        Returns:
+            list: The return value
+    """
+    return_list = list()
+    for i in array:
```
Comment (ghost) [inline tasks/kyu8.py:155]: You can use list comprehension.

Comment (OlyaKhomyn) [reply tasks/kyu8.py:155]: Yes, but it still is correct to write it this way.

**Review by LSikyr — DISMISSED** [review]

**Review by ement06 — APPROVED** [review]

**Review by khadai — APPROVED** [review]

**Review by iPavliv — APPROVED** [review]

**Pull request merged**